src/extensions/default/CSSCodeHints/main.js

```diff
@@ -374,11 +389,97 @@ define(function (require, exports, module) {
                 const propertyKey = computedPropertyKeys[resultItem.sourceIndex];
                 if(properties[propertyKey] && properties[propertyKey].MDN_URL){
                     resultItem.MDN_URL = properties[propertyKey].MDN_URL;
+                    MDN_PROPERTIES_URLS[propertyKey] = resultItem.MDN_URL;
```

Comment:
use `camelCase` vars here as the property is being mutated. "CAPS_CASE` is usually used if the value never changes during execution.